src/main/scala/gen/gmos/GmosNLongslitD.scala

```diff
@@ -29,26 +29,103 @@ import scala.concurrent.duration.FiniteDuration
 sealed trait GmosNLongslitD[F[_]] {
 
   /**
-   * Generates a sequence that is as long as necessary to achieve the desired
-   * s/n ratio.  Integration time is adjusted throughout to match conditions.
+   * Generates an acquisition sequence which terminates when the provided
+   * `acquired` effect evaluates `true`.
+   *
+   * @param itc integration time calculator
+   * @param acquired a program that can be evaluated to determine whether the
+   *                 acquisition has completed
+   *
+   * @return a sequence that acquires the target
+   */
+  def acquisition(
+    itc:      Itc[F],
+    acquired: F[Boolean]
```

Comment:
I anticipate `acquired` actually checking with the user at least initially since it is apparently hard to automate this.